spec/displaynames.html

```diff
@@ -50,7 +50,7 @@ <h1>Intl.DisplayNames ( _locales_, _options_ )</h1>
         1. If NewTarget is *undefined*, throw a *TypeError* exception.
         1. Let _displayNames_ be ? OrdinaryCreateFromConstructor(NewTarget, *"%DisplayNames.prototype%"*, &laquo; [[InitializedDisplayNames]], [[Locale]], [[Style]], [[Type]], [[Fallback]], [[Fields]] &raquo;).
         1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
-        1. Let _options_ be ? ToObject(_options_).
+        1. Let _options_ be ? GetOptionsObject(_options_).
```

Comment:
why isn’t this one also with a null prototype?

Reply:
I assume it was by mistake ... This discrepancy is exactly what GetOptionsObject will hopefully avoid.

Reply:
Right. I feel that this one was an oversight too. Thankfully, the new methods make it easier to get things right :smile:

Reply:
ah ok, the `null` one is a mistake, not this one :-)